devops/scripts/create_esrp_job.ps1

```diff
@@ -0,0 +1,87 @@
+# create_esrp_job.ps1
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+<#
+  .SYNOPSIS
+  MSFT-Internal ESRP Signing Tool
+
+  .DESCRIPTION
+  Creates an ESRP job manifest for signing production binaries (MSFT-Internal)
```

Comment:
>(MSFT-Internal) [](http://example.com/codeflow?start=63&length=15)

Same as the other comment. This needs to be clarified. Feel free to take it offline to discuss.